test/settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
This settings file has settings to be used during testing.

Each method checks for a `module.exports.debug.{name}` and outputs a message to the console for debugging purposes.

I'll update the contributing file.